### PR TITLE
Stacklog-Looker Code and Instructions

### PR DESCRIPTION
**Pull request by peterfishergcp**
Submitting the set up guide for pushing the Dialogflow Stackdriver logs to BigQuery, and then visualizing the log files with a Looker JSON Parser Block that is available on Looker Marketplace.

### CODE REVIEW

File: stacklogs-looker/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (tyhu-google) [inline stacklogs-looker/LICENSE:1]: LICENSE file is already on the top level folder. We probably don't have to put it under this folder.

File: stacklogs-looker/images/looker_marketplace/test [hunk 2]
```diff
@@ -0,0 +1 @@
+asdf
```
Comment (tyhu-google) [inline stacklogs-looker/images/looker_marketplace/test:1]: This is a redundant file I guess?

File: stacklogs-looker/images/test [hunk 3]
```diff
@@ -0,0 +1 @@
+asdf
```
Comment (tyhu-google) [inline stacklogs-looker/images/test:1]: ditto

File: stacklogs-looker/test [hunk 4]
```diff
@@ -0,0 +1 @@
+adf
```
Comment (tyhu-google) [inline stacklogs-looker/test:1]: ditto.

File: stacklogs-looker/README.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
+# stacklogs-looker
+Exporting Dialogflow Log Files to BigQuery and Visualizing Conversations with Looker
+
+
+High Level Overview
+1. Create Dialogflow Agent
+2. Create BigQuery DataSet to hold Dialogflow Stackdriver Logs and Looker Persistent Derived Table
```
Comment (tyhu-google) [inline stacklogs-looker/README.md:7]: DataSet -> Dataset

Comment (tyhu-google) [inline stacklogs-looker/README.md:7]: DataSet-> dataset

File: stacklogs-looker/README.md [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+
+![Creating the Sink Screenshot](images/image15.png)
+
+
+When creating the sink,  we limited the logs to be exported just to Dialogflow by selecting to only export those logs that equal: resource.type="global" 
+You can see that there are two log files related to Dialogflow. We would like to select ALL Logs to push both logs to BigQuery.
+
+![Creating the Sink Screenshot](images/image16.png)
+
+You can use the sink creation wizard, where you provide the sink name, the destination - BigQuery, and you can either select a BigQuery DataSet, or create a BigQuery DataSet. The Dataset is going to receive the dialogflow logs.
```
Comment (tyhu-google) [inline stacklogs-looker/README.md:76]: ditto

Comment (tyhu-google) [inline stacklogs-looker/README.md:76]: ditto

Comment (tyhu-google) [inline stacklogs-looker/README.md:76]: Dataset -> dataset

**Review by tyhu-google — COMMENTED** [review]
Thanks for preparing the code and instructions. It looks to me in general, just a few trivial comments there.
If there is a feature request or a customer reported issue for this repo, I assume you and your team is the contact to deal with that?

**Comment by peterfishergcp** [comment]
Looks good...and yes I would be the support for this.

**Comment by tyhu-google** [comment]
Peter, I saw you marked those comments as resolved, but I didn't see the change? Did you forget to push your change?

**Comment by peterfishergcp** [comment]
Thanks! I am pushing my changes now.

**Pull request closed without merging**